test/value-binding.html

```diff
@@ -21,28 +21,56 @@
 </head>
 <body>
 
-  <polymer-element name="x-foo" attributes="value" noscript>
+  <dom-module id="x-foo">
```

Comment:
I've switched basic and raw tests to test-fixtures. 

I can't figure out how to do it for value-binding.html. Problem is that I have 2 custom elements, and one element is nested, and references the other.